### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: cmake/SetupYamlCpp.cmake [hunk 1]
```diff
@@ -12,7 +12,7 @@ if (NOT yaml-cpp_FOUND)
   include(FetchContent)
   FetchContent_Declare(yaml-cpp
     GIT_REPOSITORY https://github.com/jbeder/yaml-cpp.git
-    GIT_TAG yaml-cpp-0.7.0
+    GIT_TAG yaml-cpp-0.8.0
```
Comment (knelli2) [inline cmake/SetupYamlCpp.cmake:15]: Update installation docs for this as well. Also I think the container version needs be updated as well

File: docs/Installation/Installation.md [hunk 2]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
```
Comment (knelli2) [inline docs/Installation/Installation.md:10]: I think we should keep a blurb of some kind here at the top before going in to details. The import links I think that should be highlighted are

- installation on clusters
- versioning and releases

File: docs/Installation/Installation.md [hunk 3]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
...
+
+```
+apptainer run docker://sxscollaboration/spectre bbh generate-id \
+  -q 1 --chi-A 0 0 0 --chi-B 0 0 0 -D 16 -w 0.015 -a 0 -o ./bbh_id
+apptainer run docker://sxscollaboration/spectre plot slice \
+  bbh_id/BbhVolume*.h5 -C 0,0,0 -n 0,0,1 -u 0,1,0 -X 24 24 \
+  -y ConformalFactor -o plot.pdf
+```
+
+The containers currently have precompiled code only for Linux x86 platforms.
```
Comment (knelli2) [inline docs/Installation/Installation.md]: Make this a `\note` so it's more prominent, and then also link to the Apply Silicon instructions so mac users have somewhere to look

File: docs/Installation/Installation.md [hunk 4]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
...
-\subpage spectre_build_system page. For instructions on installing SpECTRE on
-clusters please refer to the \subpage installation_on_clusters page. For
-instructions on installing SpECTRE on Apple Silicon Macs, please refer to the
-\subpage installation_on_apple_silicon page. Refer to the
-\subpage versioning_and_releases page for information on specific versions
-to install.
+### Running containerized releases
+
+A quick way to run the code without installing anything at all is with our
+containerized releases:
```
Comment (knelli2) [inline docs/Installation/Installation.md:25]: I think we need to emphasize here that the entry point to this container is the CLI (and link to the docs) and that only very specific executables are built, namely SolveXcts, BBH, SingleBH. We don't want to give people the impression that they can just download this container and run anything from spectre.

File: docs/Installation/Installation.md [hunk 5]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
...
+```
+apptainer run docker://sxscollaboration/spectre --help
+```
+
+For example, you can generate initial data for a simulation of merging black
+holes and plot the result like this:
+
+```
+apptainer run docker://sxscollaboration/spectre bbh generate-id \
+  -q 1 --chi-A 0 0 0 --chi-B 0 0 0 -D 16 -w 0.015 -a 0 -o ./bbh_id
```
Comment (knelli2) [inline docs/Installation/Installation.md:46]: [optional] I'd advise against putting these examples in since we may change arguments/names/anything at some point, and then forget to update this, and then the very first easy thing we tell people to do won't work.

File: docs/Installation/Installation.md [hunk 6]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
...
+  See \ref building-spectre for details and \ref common_cmake_flags for a list
+  of possible configuration options. For example, set `-D ENABLE_OPENMP=ON`
+  to enable OpenMP-parallelization for the exporter library.
+
+  Now you can compile executables (again, see \ref building-spectre for
+  details). For example:
+
+  ```sh
+  make -j12 cli
+  make -j12 BundledExporter
```
Comment (knelli2) [inline docs/Installation/Installation.md:142]: Probably do `-j4` just to lessen the likelihood that people will run out of memory

File: docs/Installation/Installation.md [hunk 7]
```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
...
+  ```
+
+### Installation with Spack
+
+You can also install SpECTRE with the [Spack](https://github.com/spack/spack)
+package manager:
```
Comment (knelli2) [inline docs/Installation/Installation.md:148]: Do we want to warn people that this may not work for a number of reasons? (local env, spack upstream, etc.)

**Review by knelli2 — CHANGES_REQUESTED** [review]
You can squash these in directly

File: docs/Installation/Installation.md (hunk 3 above)
Comment (wthrowe) [inline docs/Installation/Installation.md]: Drive-by comment, no need for further input from me: To many Linux users "x86" suggests 32-bit.  Using "x86_64" is unambiguous.

File: cmake/SetupYamlCpp.cmake (hunk 1 above)
Comment (nilsvu) [reply cmake/SetupYamlCpp.cmake:15]: No v0.7.0 is still fine, this just bumps the version that is fetched automatically.

File: docs/Installation/Installation.md (hunk 4 above)
Comment (nilsvu) [reply docs/Installation/Installation.md:25]: Currently only CLI, CCE, and SolveXcts are built

File: docs/Installation/Installation.md (hunk 5 above)
Comment (nilsvu) [reply docs/Installation/Installation.md:46]: I thought it would be nice to give an example that actually does something, but I agree with your concern. I'll revisit this when I put the initial data tutorial online and maybe just link to it.

File: cmake/SetupYamlCpp.cmake (hunk 1 above)
Comment (knelli2) [reply cmake/SetupYamlCpp.cmake:15]: Oh right, my bad

File: docs/Installation/Installation.md [hunk 8]
```diff
@@ -6,19 +6,142 @@ See LICENSE.txt for details.
...
+
+The containers currently have precompiled code only for Linux x86_64 platforms,
+and only have a limited set of executables precompiled. The supported features
+available in the precompiled containers are:
+
+- Generating initial data
+- Running CCE (see \ref tutorial_cce)
+- Running Python support code with the SpECTRE CLI (see \ref tutorial_cli)
+
+### Running precompiled executables
```
Comment (knelli2) [inline docs/Installation/Installation.md]: Given the wording in the previous section, this title is a bit confusing since this section is completely unrelated to the docker container. Maybe something like "Running with static binaries" ?

**Review by knelli2 — CHANGES_REQUESTED** [review]
Looks good, just one last thing. You can squash everything together

**Review by knelli2 — APPROVED** [review]
